.eslintrc.json

```diff
@@ -43,36 +43,51 @@
     },
     { "files": ["**/__tests__/**"], "env": { "jest": true } },
     {
+      "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/stylistic"
+      ],
```

Comment:
Note that this doesn't enable type checked rules internally. I tried and they OOMd (https://github.com/typescript-eslint/typescript-eslint/issues/1192). 😞 

As a followup I'd love to try tinkering with the monorepo configuration (https://typescript-eslint.io/linting/typed-linting/monorepos) and/or using our new `EXPERIMENTAL_useProjectService` option (https://github.com/typescript-eslint/typescript-eslint/pull/6754)